src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000035> <http://purl.obolibrary.org/obo/ENVO_00000020>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000035> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0001025> <http://purl.obolibrary.org/obo/ENVO_00000133>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000036> (sedimentation in a water body)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000036> "A material accumulation process during which solid particles are pulled through a water body by gravitation or centrifugal force and which ends when they settle on a solid surface.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000036> "sedimentation in a water body"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000036> <http://purl.obolibrary.org/obo/ENVO_03000009>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000036> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01000875>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000036> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000063>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000036> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01000060>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000036> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002229> <http://purl.obolibrary.org/obo/ENVO_03000010>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000037> (ice ablation zone)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000037> "An ablation zone in which an ice ablation process occurs.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000037> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000037> "ice ablation zone"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000037> <http://purl.obolibrary.org/obo/ENVO_01000914>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000037> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000067> <http://purl.obolibrary.org/obo/ENVO_01000919>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000038> (cryoconite)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://www.nature.com/nrmicro/journal/v13/n11/full/nrmicro3522.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cryoconite") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000038> "Particulate matter including dust, rock, soot, and microorganisms deposited on an ice mass by an aeolian process.")
```

Comment:
@pbuttigieg definition updated. 

as for microbes being essential for cryoconite, it seems that microbes a part of the [cryoconite mixture](http://www.aber.ac.uk/en/news/archive/2013/01/title-125754-en.html).